frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -135,7 +135,6 @@ class StorageClassDropdown_ extends React.Component<StorageClassDropdownProps, S
             title={this.state.title}
             onChange={this.onChange}
             id={id}
-            menuClassName="dropdown-menu--text-wrap"
```

Comment:
Does removing this class cause problems when the text *does* wrap? What if I add a long description to the storage class?

cc @rhamilto

Reply:
I guess I don't have an idea of how long it could get - right now it just gets longer and longer on one line. It seems the description is the concatenation of 5 pieces of information and they all seemed like they wouldn't each be too long because they are not free text fields - just the (autogenerated) description, accessMode, provisioner, type and zone.

Reply:
We need to make sure at some point we wrap. I wouldn't assume it's restricted to a certain length. On a mobile device, this will likely wrap as well.

Reply:
Removing this class will mean the text won't ever wrap.